app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    total = 0
+
+    for player in players:
```

Comment:
Use list comprehension

Reply:
I can agree with you here ```sum(player.get_rating() for player in players)```. Cause it's not the lsit comprehension, it's more about generators.